collect-modified-images.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Collect the names of all images
+  hosts: localhost
+  become: false
+  gather_facts: true
+
+  vars:
+  environment:
+    PATH: "{{ ansible_env.PATH }}:{{ ansible_env.HOME }}/.local/bin"
+      # when: run_upstream|bool
```

Comment:
Should this be un-commented or deleted?

Reply:
This file should actually be deleted come to think of of it